src/NoFrixion.MoneyMoov/Extensions/JsonExtensions.cs

```diff
@@ -42,7 +42,7 @@ private static string ToJsonFlatSystemText<T>(this T obj, bool usePascalCase = f
     private static string ToJsonFormattedSystemText<T>(this T obj, bool usePascalCase)
         => System.Text.Json.JsonSerializer.Serialize(obj, MoneyMoovJson.GetSystemTextSerialiserOptions(true, usePascalCase: usePascalCase));
 
-    private static T? FromJsonSystemText<T>(this string json, bool usePascalCase)
+    public static T? FromJsonSystemText<T>(this string json, bool usePascalCase)

```

Comment:
Oooh, do you really want this? It will open a bit of a can of worms. We're trying to standardise on one JSON library and Newtonsoft is currently the most suitable choice https://nofrixion.atlassian.net/wiki/spaces/NDS/pages/120193025/JSON+Serialisation

Reply:
I'll revert that. The code isn't using it.